.github/workflows/releaseTrigger.yml

```diff
@@ -0,0 +1,46 @@
+# This workflow will build a Java project with Gradle
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-gradle
+
+name: Release
+
+on:
+  create:
+    tags:
+      - 'releaseTrigger'
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    name: releasing
+    steps:
+      - name: Wait for build to succeed
+        uses: jitterbit/await-check-suites@v1
+        with:
+          appSlugFilter: 'github-actions'
+          timeoutSeconds: '300'
+      - uses: actions/checkout@v2
+      - name: Setup java
+        uses: actions/setup-java@v1
+        with:
+          java-version: 8
+      - name: Perform Release
+        uses: eskatos/gradle-command-action@v1
+        env:
+          TRAVIS_BRANCH: master
+          TRAVIS_COMMIT_MESSAGE: <no commit message, not needed, we do not care about them, if we trigger or not>
+          TRAVIS_PULL_REQUEST: false
+          GH_WRITE_TOKEN: ${{ secrets.GH_WRITE_TOKEN }}
+          BINTRAY_USER_NAME: ${{ secrets.BINTRAY_USER_NAME }}
+          BINTRAY_API_KEY: ${{ secrets.BINTRAY_API_KEY }}
+          NEXUS_TOKEN_USERNAME: ${{ secrets.NEXUS_TOKEN_USERNAME }}
+          NEXUS_TOKEN_PASSWORD: ${{ secrets.NEXUS_TOKEN_PASSWORD }}
+        with:
+          arguments: ciPerformRelease
+      - name: Delete Tag
+        run: git push --delete origin releaseTrigger
```

Comment:
This means a broken release requires manual cleanup of the tag. Would that be a reason to perform this step much earlier, i.e. as the first one?

Reply:
that is a really good question -> we also should think ob some kind of notification in the error case. Or we can also provide a flag, which will continue the job, even if it failed. That is something hard to tell, which we should define. I do not have any special preferences regarding that :)

Reply:
I think it's bad enough that we abuse tags this way, we shouldn't go further and build an entire protocol based on them. :wink: The check marks next to each commit will show how the build went. Additionally, you can configure Bintray to send you an email for each release.

Reply:
I think we can use this feature:

https://help.github.com/en/actions/reference/context-and-expression-syntax-for-github-actions#job-status-check-functions

Per default, steps only run "when none of the previous steps have failed or been canceled". Using `if: ${{ always() }}` seems to be a good fit for clean-up tasks like this.

(Sorry, haven't found time for a full review so far.)

Reply:
> Per default, steps only run "when none of the previous steps have failed or been canceled". Using `if: ${{ always() }}` seems to be a good fit for clean-up tasks like this.

added

Reply:
@aepfli could it be that you forgot the condition in your commit 254583d?

Reply:
you are right, sorry my bad -.-